client/schema/v0/index.js

```diff
@@ -351,16 +350,15 @@ export default {
               }
             ]
           },
-          animals: {
+          'legacy-animals': {
             title: 'Type of animals',
             repeats: 'species',
             fields: [
               {
                 name: 'speciesId',
                 label: 'Animal type',
-                type: 'select',
-                className: 'smaller',
-                options: LEGACY_SPECIES
+                type: 'legacy-species-selector',
+                className: 'smaller'
```

Comment:
Options has disappeared because it's now handled inside the component, right?

Reply:
yeah